imap_processing/ultra/l2/ultra_l2.py

```diff
@@ -296,4 +345,41 @@ def ultra_l2(
 
     # Add the defined attributes to the map's global attrs
     map_dataset.attrs.update(map_attrs)
+
+    # Add the "label" coordinates to the map dataset
+    for coord_var, coord_data in map_dataset.coords.items():
+        if coord_var != "epoch":
+            map_dataset.coords[f"{coord_var}_label"] = xr.DataArray(
+                coord_data.values.astype(str),
+                dims=[
+                    coord_var,
+                ],
+                name=f"{coord_var}_label",
+            )
+
+    # Add variable specific attributes to the map's data_vars and coords
+    for variable in map_dataset.data_vars:
+        # Skip the subdivision depth variables, as these will only be
+        # present for debugging purposes
+        if "subdivision_depth" in variable:
+            continue
+
+        # The longitude and latitude variables will be present only in Healpix tiled
+        # map, and, as support_data, should not have schema validation
+        map_dataset[variable].attrs.update(
+            cdf_attrs.get_variable_attributes(
+                variable_name=variable,
+                check_schema=(
+                    True if variable not in ["longitude", "latitude"] else False
+                ),
```

Comment:
You can let the bool fall out of your computation
```suggestion
                check_schema=variable not in ["longitude", "latitude"],
```

Reply:
Good point!